bindings/kepler.gl-jupyter/README.md

```diff
@@ -14,31 +14,61 @@ Table of contacts
 
 ## Installation
 
-### Prerequisites
-- Python >= 2
-- ipywidgets >= 7.0.0
+[![Anaconda-Server Badge](https://anaconda.org/conda-forge/keplergl/badges/version.svg)](https://anaconda.org/conda-forge/keplergl) [![PyPI version](https://badge.fury.io/py/keplergl.svg)](https://badge.fury.io/py/keplergl)
 
+### Using conda:
 
-To install use pip:
+```shell
+conda install -c conda-forge keplergl
+```
+
+#### Prerequisites
+  - Python >= 3.7
 
-    $ pip install keplergl
+### Using pip:
+
+```shell
+pip install keplergl
+```
 
+#### Prerequisites
+  - For Kelplergl <= 0.3.0
+    - Python >= 2
+    - ipywidgets >= 7.0.0
+
+### Installation Issues:
+
+#### For Jupyter Notebook
 
 If you on Mac used `pip install` and running Notebook 5.3 and above, you don't need to run the following
 
-    $ jupyter nbextension install --py --sys-prefix keplergl # can be skipped for notebook 5.3 and above
+```shell
+jupyter nbextension install --py --sys-prefix keplergl # can be skipped for notebook 5.3 and above
 
-    $ jupyter nbextension enable --py --sys-prefix keplergl # can be skipped for notebook 5.3 and above
+jupyter nbextension enable --py --sys-prefix keplergl # can be skipped for notebook 5.3 and above
+```
+
+#### For JupyterLab
+
+Note: to work with JupyterLab3, you need to install KeplerGL > 0.3.0. Otherwise, you will get the following error message:
```

Comment:
does this mean no additional steps for Jupyterlab3 if KeplerGl > 0.3.0?

Reply:
yes, this should be taken care by using `jupyter_packaing`.

Reply:
@heshan0131 I've tested keplergl with JupyterLab 1/2/3, and here is the updated documentation for installation with JupyterLab. Let me know any questions and issues. Thanks!

### KeplerGL with JupyterLab 1 & 2 & 3

## 1. For JupyterLab 3

NOTE: `KeplerGL` <=0.3.0 doesn't work with JupyterLab 3. You need to make sure the python package `KeplerGL` > 0.3.0 is installed. 

Installation using pip:
```shell
pip install keplergl
```
Installation using conda:
```shell
conda install keplergl
```

There is no need to use `jupyter labextension install` for `KeplerGL` > 0.3.0 with JupyterLab3.

## 2. For JupyterLab 1

For JupyterLab1, you need to install `keplergl-jupyter` labextension from NPM registery. There is no need to install `KeplerGL` python package. 

First, install `jupyterlab-manager` for JupyterLab1:
```shell
jupyter labextension install @jupyter-widgets/jupyterlab-manager@1.1
```
NOTE: only @jupyter-widgets/jupyterlab-manager@1.1 (version 1.1) works with JupyterLab1.

Then, install `keplergl-jupyter` labextension from NPM registry:
```shell
jupyter labextension install @jupyter-widgets/jupyterlab-manager@1.1 keplergl-jupyter
```

## 3. For JupyterLab 2

For JupyterLab2, you need to install `keplergl-jupyter` labextension from NPM registery. There is no need to install `KeplerGL` python package. 

First, install `jupyterlab-manager` for JupyterLab2:
```shell
jupyter labextension install @jupyter-widgets/jupyterlab-manager@2
```

To install `keplergl-jupyter` from NPM registry, JupyterLab2 has following requirements of dependencies:
> JupyterLab             Extension               Package
> '>=16.9.0 <16.10.0'   >=17.0.0 <18.0.0  react
> '>=16.9.0 <16.10.0'  >=17.0.0 <18.0.0  react-dom

However, `keplergl-jupyter`<=0.3.0 depends on react >= 17.0.2. Therefore, the latest `keplergl-jupyter` can’t be installed with JupyterLab2: if you use `jupyter labextension install keplergl-jupyter`, the version 0.2.2 as a fallback will be installed. Unfortunately, version 0.2.2 doesn’t work with JupyterLab2.

A workaround is to modify the file `lib/python3.8/site-packages/jupyterlab/staging/package.json` and remove “react” and “react-dom” from “singletonPackages” object. Then, install keplergl-jupyter using this command:
```
jupyter labextension install keplergl-jupyter
```